README.md

```diff
@@ -1,18 +1,116 @@
+[![Build Status](https://travis-ci.org/UCLA-CS130/Caviar-Hacker-Grid.svg?branch=master)](https://travis-ci.org/UCLA-CS130/Caviar-Hacker-Grid)
+
 # Caviar-Hacker-Grid
 CS130 Winter 2017 Webserver Project
 
 Commands are to be run from the base Caviar-Hacker-Grid directory.
+# Installation
+You must clone the repository recursively.
+``` 
+git clone --recursive https://github.com/UCLA-CS130/Caviar-Hacker-Grid.git 
+```
```

Comment:
It's literally out of context.
There's no reason to handle exception cases where users are bad at reading.